### PR TITLE
Migrate from requires.io to dependabot

### PR DESCRIPTION
**Pull request by niklasmohrin**
Although we didn't notice, our badge has been linking to [the deprecation post of requires.io](http://shiningpanda.com/requires-io-clap-de-fin/) for some time now. Dependabot seems convenient.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 1
+
```
Comment (janno42) [inline .github/dependabot.yml:8]:
```suggestion
    open-pull-requests-limit: 5
    labels:
      - "[T] Dependencies"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 1
```
Comment (janno42) [inline .github/dependabot.yml:13]:
```suggestion
    open-pull-requests-limit: 5
    labels:
      - "[T] Dependencies"
```

**Review by janno42 — DISMISSED** [review]

**Comment by He3lixxx** [comment]
Since we're at it here: Dependabot adds new labels to the repository when creating PRs. For https://github.com/e-valuation/EvaP/pull/1844, it added `javascript`. Can we disable that? I'm fine if it just doesn't tag its PRs.

**Comment by niklasmohrin** [comment]
yup, todo for me: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#labels

**Comment by niklasmohrin** [comment]
I think with the explicit `labels`, it should now not add any new labels anymore

**Review by He3lixxx — APPROVED** [review]
thanks :)

**Pull request merged**